README.md

```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
-| applications.<application_name>       | `endpoint`: AI DIAL Application API for chat completions.<br />`iconUrl`: Icon path for the AI DIAL Application on UI.<br />`description`: Brief AI DIAL Application description.<br />`displayName`: AI DIAL Application name on UI.<br />`inputAttachmentTypes`: A list of allowed MIME types for the input attachments.<br />`maxInputAttachments`: Maximum number of input attachments (default is zero when `inputAttachmentTypes` is unset, otherwise, infinity) <br/> `forwardAuthToken`: If flag is set to `true` forward Http header with authorization token to chat completion endpoint of the application.                                                                                                                                                                                                                                                                                                                                                                                                                          |
-| models                                | A list of deployed models and their parameters:<br />`<model_name>`: Unique model name.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Parameter                             | Description      |
+|---------------------------------------|------------------------------------------------------------------------------------------|
+| routes | Path(s) for specific upstream routing or to respond with a configured body.   |
```

Comment:
Add example to the sample file

Reply:
We do not have here Addons section

Reply:
We do not have Assistant section here

Reply:
@astsiapanay  to check wordings in new descriptions

Reply:
fixed